tabbycat/participants/models.py

```diff
@@ -129,6 +130,9 @@ def __str__(self):
         return str(self.name)
 
     def get_public_name(self, tournament):
+        # This allows for jinja html templates to still access redactions from the tab
+        if self.anonymous:
+            return mark_safe("<em>Redacted</em>")
```

Comment:
"Redacted" should be marked for translation, like
```suggestion
            return mark_safe("<em>" + _("Redacted") + "</em>")
```
No need for that comment (it's also wrong as we don't use Jinja) :P